sqlite_utils/db.py

```diff
@@ -3011,6 +3048,85 @@ def truncate(value):
             least_common,
         )
 
+    def add_geometry_column(
+        self,
+        column_name: str,
+        geometry_type: str,
+        srid: int = 4326,
+        coord_dimension: str = "XY",
+        not_null: bool = False,
+    ) -> bool:
+        """
+        In Spatialite, a geometry column can only be added to an existing table.
```

Comment:
Tiny style thing: `Spatialite` here should be `SpatiaLite` for consistency with other documentation.

Reply:
Got it.